### ISSUE TITLE
Commting version

### ISSUE THREAD

**Issue by nivekcode** [kreuzerk/release-me-daddy#4]

**Comment by maquejp** [comment]
Can you update this? I am wandering how to (I have following your YTB)
I suppose this is a this level in the top package.json

"bump-version": "rjp package.json version $VERSION && rjp projects/eui-table/package.json version $VERSION"

**Comment by nivekcode** [comment]
yes exactly. you run "bump-version" during the release. Therefore you have to add the following config to the semantic release config

```
 ["@semantic-release/exec", {
      "prepareCmd": "VERSION=${nextRelease.version} npm run bump-version"
    }]
```

**Comment by maquejp** [comment]
Will it increment the version automatically?
